src/main/java/io/supertokens/authRecipe/AuthRecipe.java

```diff
@@ -205,12 +206,15 @@ private static void deleteNonAuthRecipeUser(AppIdentifierWithStorage
         appIdentifierWithStorage.getUserRolesStorage()
                 .deleteAllRolesForUser(appIdentifierWithStorage, userId);
```

Comment:
when u merge the latest changes from the multi-tenant-config branch, you will see that this part has changed slightly. So please sync up

Reply:
Not sure what do you mean.

Reply:
See deleteNonAuthRecipeUser function